bf2fc6cc711aee1a0c2a/kas-fleet-manager
# fix(route-template.yml): removed non-working certs from template to allow for auto creation of certs

## Description


The removal of the certs from the template allows openshift to auto create functional certs that allow for communication between kas fleetshard sync and the fleet manager

closes issue MGDSTRM-3506

## Verification Steps


1. Deploy Kas fleet manager to an OpenShift Cluster.
2. Attempt to hit the route address with a curl command like 
`curl -H "Authorization: Bearer $(ocm token)" <route-location>/api/managed-services-api/v1/kafkas`

 - or any other endpoint.
3.  You should get a response containing 
`curl: (60) SSL certificate problem: self signed certificate`

4. Edit the route and remove cert and key, save and reload
5. Now an attempt to hit the api endpoint again. It should be successful
## Type of change


## Checklist (Definition of Done)

~~- [ ] Unit and integration tests added that prove the fix is effective or the feature works (tested against emulated and non-emulated OCM environment)~~
~~- [ ] Documentation added for the feature~~
~~- [ ] Required metrics/dashboards/alerts have been added (or PR created).~~
~~- [ ] Required Standard Operating Procedure (SOP) is added.~~
~~- [ ] JIRA has created for changes required on the client side~~
